src/components/selectActivity/selectActivity.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { useState } from "react";
+
+import { ISelectActivityProps } from "types/common/ComponentsProps";
+
+import styles from "./selectActivity.module.scss";
+
+const SelectActivity = ({ selected, onChange, onClose }: ISelectActivityProps) => {
+	const [localSelected, setLocalSelected] = useState<string[]>(selected);
+
+	const activityList = [
+		"UX/UI Design",
+		"Mobile App",
+		"Web Design",
+		"Graphic Design",
+		"Logo creating",
+		"Video editing",
+	];
+
+	// eslint-disable-next-line no-undef
+	const handleCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+		const value = e.target.value;
+		setLocalSelected((prev) =>
+			e.target.checked ? [...prev, value] : prev.filter((item) => item !== value),
+		);
+	};
+
+	// eslint-disable-next-line no-undef
+	const handleApply = (e: React.MouseEvent<HTMLButtonElement>) => {
+		e.preventDefault();
+		onChange(localSelected);
+		onClose();
+	};
```

Comment:
better remove eslint disabling and make import ChangeEvent and MouseEvent from react